backend/server/src/server.rs

```diff
@@ -288,6 +297,106 @@ impl AdServer {
         }
     }
 
+    fn get_advertisers(&self, req: Request) -> ResponseFuture {
+        if let Some(lang) = AdServer::get_lang_from_headers(req.headers()) {
+            let options = HashMap::new();
+            let db_pool = self.db_pool.clone();
+            let pool = self.pool.clone();
+
+            fn advertiser_count_getter(a: &Advertisers) -> i64 {
+                a.count
+            }
+            let top_advertisers = spawn_with_clone!(pool; lang, db_pool, options; 
+                                                        Advertisers::get(&lang, &db_pool, &options, Some(1000)));
+            let future = top_advertisers.map(|advertisers| 
+                advertisers
+                .into_iter()
+                .filter(|advertiser| advertiser_count_getter(&advertiser) > 10 )
```

Comment:
You can do advertiser.count as i64 here I think

Reply:
Also I find it weird that it looks like we are filtering database output. Are we sure that we can’t handle this in a query?

Reply:
Oh, yeah, you can't do HAVING in Diesel.

Reply:
(So confirmed the dudes in the Diesel gitter chat)